docs/extend/plugins.md

```diff
@@ -53,6 +53,10 @@ The following plugins exist:
   another volume plugin that provides multi-host volumes management for Docker
   using GlusterFS.
 
+* The [Horcrux Volume Plugin](https://github.com/muthu-r/horcrux)
+  An On Demand, Version controlled access to your Data. Horcrux is an open-source plugin,
+  written in Go, and supports SCP, [Minio] (https://www.minio.io), and Amazon S3.
```

Comment:
Looks like the minio link has a space between `]` and `(`, making it not work as link.